src/pages/en/reference/api-reference.md

```diff
@@ -701,6 +701,149 @@ Pagination will pass a `page` prop to every rendered page that represents a sing
 | `page.url.prev`    | `string \| undefined` | Get the URL of the previous page (will be `undefined` if on page 1).                                                              |
 | `page.url.next`    | `string \| undefined` | Get the URL of the next page (will be `undefined` if no more pages).                                                              |
 
+## Content Collections (Experimental)
+
+<p>
+  <Since v="1.7.0" />
+</p>
+
+Content collections offer APIs to configure and query your Markdown documents in `src/content/`. For feature rundowns and usage examples, [see our content collections guide](/en/guides/content-collections/).
+
+### Collection Entry Type
```

Comment:
question: Does it maybe make sense to \show these closer to the order in which someone might encounter/use them?

It almost feels like burying the lede a bit to save "defining a collection" to the very end? I might be tempted to go `defineCollection()` first. Then I'd probably think of `getCollection()` and `getEntry()` first, before the the Collection Entry Type? That way, we also don't have the reference of "things below yet to come" in this first line. That is often a sign that maybe things should go in a different order.

I know for this reference page it's not as important as our Guide page in docs. But, I still kind of like organizing these in the order someone might use, and therefore understand, them. Someone *might* be reading these in order, and that seems like a logical flow to me.

Reply:
Agreed! Reordering to:
1. `defineCollection`
2. `getCollection` and `getEntry`
3. Collection type